src/parser/jahia_site.py

```diff
@@ -132,6 +139,88 @@ def parse_properties(self):
 
         self.server_name = properties["siteservername"]
 
+    def parse_menu(self):
```

Comment:
Cette méthode possède 7 boucles for imbriquées ... Pas facile à relire ... je te fais donc confiance et j'espère ne pas à avoir à mettre les mains dedans :-)